packages/editor/src/components/post-discussion/panel.js

```diff
@@ -86,6 +86,16 @@ function PostDiscussionToggle( { isOpen, onClick } ) {
 	);
 }
 
+/**
+ * Uses the `useSelect` hook to determine if the discussion panel is enabled.
```

Comment:
I think we should have a higher level small description of the component and not describe its current implementation, with maybe the exception of mentioning the panel name that can be used to programmatically hide the panel. Something like:

> This component allows to update comment and pingback settings for the current post. Internally there are checks whether the current post has support for the above and if the `discussion-panel` panel is enabled.

Reply:
Agreed. The implementation details aren't crucial for similar documents.